Source/Private/CommandCoreResult.ps1

```diff
@@ -0,0 +1,6 @@
+class CommandCoreResult {
```

Comment:
Probably best to avoid adding a class because of the issues with Powershell's type system. Either that, or we should find a way to make sure the types are easily imported when using our module.

Reply:
Thought about that, but the command does not set `$Success`. Can't really share the same class either.

There were earlier separations of `$StdOut` and `$StdErr` but decided to follow your practice of just `$Output.